.vscode/redux.code-snippets

```diff
@@ -1,4 +1,11 @@
 {
+    "better rebuild autocomplete": {
```

Comment:
I'm hoping you'll let me sneak this in, it's just a snippet to save time because the rebuild function takes a function from AppStateBuilder to dynamic and the autocomplete doesn't seem to know how to deal with that and gives you `.rebuild(() => null)` which you then have to go back and change.